tests/lax_numpy_test.py

```diff
@@ -2119,6 +2119,19 @@ def testCov(self, shape, dtype, rowvar, ddof, bias, rng_factory):
     self._CompileAndCheck(lnp_fun, args_maker, check_dtypes=True, atol=tol,
                           rtol=tol)
 
+  def test_convolve(self):
+    x = onp.array([1.,2.,3.])
```

Comment:
Can you test multiple sizes of `x` and `y`? You would probably do well to make this a parametric test, as many others in this file are.

You could also avoid redundancy in the test by making the `mode` argument parametric as well.